v22.1/security-reference/transport-layer-security.md

```diff
@@ -0,0 +1,280 @@
+With asymmetric encryption, by contrast, each party can share their public key, allowing anyone to send them messages that only they can decrypt with their private key. In the context of internet communication
+
+In the TLS protocol, asymmetric encryption using the public/private key pairs is used to securely exchange random data from that is used to create shared **session keys**, which are then used by both parties to (symmetrically) encrypt and decrypt data for the remainder of the session. Since a session key is created with information shared using asymmetric encryption, and is used only for the duration of a session, TLS combines the best of both asymmetric encryption (the ability to establish an encrypted communication channel without having to have first securely shared a key) and symmetric encryption (computational efficiency).
+
+{{site.data.alerts.callout_info}}
+The generation and management of session keys is fully automated within the TLS protocol. You do not ever need to provision or manage TLS session keys.
+{{site.data.alerts.end}}
+
+## What is Public Key Infrastructure (PKI)?
+
+Encryption is powerful and important, but without identity authentication, it's not very useful: if you don't know who is sending you messages in first place, it's small comfort that nobody *else* is tampering with the contents or eavesdropping on the conversation.
+
+If you encrypt a  message with a TLS public key, you know that only a holder of the matching private key will be able to decrypt it. And perhaps the holder of the public certificate identifies themself as your friend, your bank, your employer, or a government. But how can you trust that the certificate was ever actually held by the party you want to reach, rather than an imposter?
+
+This is a complex social and technological problem, and the primary solution used today by the internet as a whole, as well as organizations across the world, is mix of careful practices, social relationships, and digital technologies. It is what is known as Public Key Infrastructure (PKI).
+
+At its core, PKI is a hierarchy of cryptographically backed trust relationships between the following categories of interested party:
+
+- **Subscribers** wish to use the PKI to prove their identity to the world and offer secure services to others.
+- **Relying parties** wish to connect with subscribers, secure in the their knownledge of the identity of the latter.
+- **Certificate Authorities (CA)** are responsible for verifying the identity of subscribers (which can include subornidate certificate authorties).
+
+### Certificates, signing, trust and authority
+
+The core mechanism of PKI is a the PKI certificate, also known as a 'security certificate', 'digital certificate' or 'TLS certificate' (because it is used in TLS), or abbreviated "cert".
+
+A PKI certificate is a file containing the following:
+
+- A) A public key to be used for [key pair encryption, often including TLS](#what-is-transport-layer-security-tls).
```

Comment:
I'd make this an ordinal list and set the first bullet to be `a.` instead of `1.`.